### PR TITLE
26-BUNSPACE2024-StarWars-Text-Effect-Lore-Page

### LINKED ISSUE #26: Add Star wars effect text in lore page
Add basic text, a lot of it to the lore page and allow it like this
https://starwarsintrocreator.kassellabs.io/#!/Episode9

Skip the intro of blue text but keep the star wars logo fadding away and the text from the bottom rising up

### PR DESCRIPTION
**Pull request by GrpFruit**
add starwars effect in lore page
Add fade
add reference of text before effect

## Extra Resources

- [The 10 commandments of navigating code reviews](https://techbeacon.com/app-dev-testing/10-commandments-navigating-code-reviews)
- [Google's Code Review Developer Guide](https://google.github.io/eng-practices/review/)

Closes #26

### CODE REVIEW

File: ui/src/pages/Lore/lore.scss [hunk 1]
```diff
@@ -1,17 +1,76 @@
...
+
+// .lore {
+//   display: flex;
+//   flex-direction: column;
+//   align-items: center;
+//   &__content {
+//     @extend .lore;
+//     width: 80%;
+//   }
+// }
```
Comment (jmorales8) [inline ui/src/pages/Lore/lore.scss:76]: remove comments and add new line plz

File: ui/src/pages/Lore/lore.scss [hunk 2]
```diff
@@ -1,17 +1,76 @@
...
+    rgba(238, 203, 203, 0.25),
+    rgba(238, 203, 203, 1),
+    rgba(238, 203, 203, 1)
+  );
+  z-index: 10;
+}
+
+body {
+  overflow: hidden;
+}
```
Comment (jmorales8) [inline ui/src/pages/Lore/lore.scss:45]: do we need to have body defined like this?

File: ui/src/pages/Lore/Lore.tsx [hunk 3]
```diff
@@ -1,20 +1,40 @@
...
+                neque, enim laborum.
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
-    </div>
+    </>
   )
 }
```
Comment (jmorales8) [inline ui/src/pages/Lore/Lore.tsx:40]: new line at the end of here plz

File: ui/src/pages/Lore/Lore.tsx [hunk 4]
```diff
@@ -1,20 +1,40 @@
...
-
-          Enim ut sem viverra aliquet eget. Ullamcorper a lacus vestibulum sed arcu non odio euismod. Non consectetur a erat nam at lectus urna duis. Mattis rhoncus urna neque viverra. Lobortis mattis aliquam faucibus purus in massa tempor nec. Volutpat maecenas volutpat blandit aliquam etiam erat velit. Lectus nulla at volutpat diam. Porta nibh venenatis cras sed felis eget. In hac habitasse platea dictumst. Augue ut lectus arcu bibendum at varius vel pharetra vel. Facilisis sed odio morbi quis commodo odio. Eget mi proin sed libero enim sed faucibus. Aliquam faucibus purus in massa tempor nec feugiat. Id aliquet risus feugiat in. Volutpat sed cras ornare arcu dui vivamus arcu felis. Pharetra sit amet aliquam id. Congue nisi vitae suscipit tellus. Quis eleifend quam adipiscing vitae proin sagittis nisl rhoncus. Tristique senectus et netus et.
-        </p>
+    <>
+      {/* <div id="fade"></div> */}
```
Comment (jmorales8) [inline ui/src/pages/Lore/Lore.tsx:4]: remove this please

File: ui/src/pages/Lore/lore.scss (hunk 2 above)
Comment (GrpFruit) [reply ui/src/pages/Lore/lore.scss:45]: I think this is a more efficient way of hiding overflow

Comment (jmorales8) [reply ui/src/pages/Lore/lore.scss:45]: I deleted this and it seemed to work the same

Comment (GrpFruit) [reply ui/src/pages/Lore/lore.scss:45]: fuck out

Comment (GrpFruit) [reply ui/src/pages/Lore/lore.scss:45]: ok ok ill remove it lolol

**Review by jmorales8 — APPROVED** [review]

**Pull request merged**